### DISCUSSION TITLE
Oracle Linux and Docker

### DISCUSSION THREAD

**Discussion by ephemeric** [Security-Onion-Solutions/securityonion#13815] [General] [answered]
Hi,

Last I installed SO was years back...

I see we've moved to OL9.4 and Docker.

Is there a writeup anywhere that explains the choices made as to why?

I'm curious as Docker is not a 1st class citizen in RHEL-like distros. Why not Podman?

Cheers.

**Comment by dougburks** [comment, marked as answer]
We moved to Docker several years ago. Since then, we've looked at Podman many times but it's never fully supported all of the Docker features that we need.

For more information about the base OS, please see:
https://blog.securityonion.net/2023/07/security-onion-24-base-os.html
